BaseImage.py

```diff
@@ -58,8 +58,10 @@ def __init__(self, fname, fname_outdir, params):
         self["dir"] = os.path.dirname(fname)
 
         self["os_handle"] = openslide.OpenSlide(fname)
+        self["image_base_size"] = self["os_handle"].dimensions
         self["image_work_size"] = params.get("image_work_size", "1.25x")
         self["mask_statistics"] = params.get("mask_statistics", "relative2mask")
+        self["base_mag"] = getMag(self, params)
```

Comment:
you shouldn't have to call getMag twice, especially since this may be an expensive function in the future. also, if folks write buggy code and you end up with 2 diference base_mag values you're life gets complicated

instead keep line 64, and use the resulting "S" value for like 65